### ISSUE TITLE
Could not reproduce similar result on SYNTH dataset

### ISSUE THREAD

**Issue by yx-chan131** [JaihyunKoh/BNUDC#3]
Hi, I follow your implementation details to train BNUDC on SYNTH dataset. I trained the model for 40 epochs and batch size of 2. The final PSNR of my model is around 45.3dB which is still 0.4dB lower than your paper's result. The deflare ability of my model is worse, the flares with longer tail still remained in my output image. Could you give some suggestion on how to improve the model training procedure? 

Have you ever tried implementing GAN loss in model training? If yes, I would like to know if it helps in generalization and output's aesthetic.

Thanks!

**Comment by JaihyunKoh** [comment]
If you have a lack of PSNR score, I recommend training more with small learning late such as 1e10-6 (optimal lr should be explored empirically). As we discussed in our paper, the remaining flare artifact in the images severely degraded by strong light  is still problem, even if you improve the model to reach the PSNR score we reported, the visual results may be similar to before, actually at the level of more than 45dB PSNR, human usually can not distinguish the difference. To improve the perceptual restoration quality, a generative approach can be a good option as shown in the SRGAN paper, but because the scenes in the UDC datasets are not much diverse, the overfitting might be another problem. As another trial, we can use the perceptual distance metrics (e.g. LPIPS, DISTS) as loss functions for suppressing the flare artifact perceptually. Visual suppression of flares is strongly demanded in the industry, improving this would be a great contribution.
